Contact us section

zzhikai opened the pull request:
Reference from FTS Month as well as FTS Website
Mobile View:
![Screenshot 2022-02-12 at 9 41 56 AM](https://user-images.githubusercontent.com/77470406/153691382-102de7d9-f0e7-409c-910f-17919d33f8f1.png)

Desktop View:
![Screenshot 2022-02-12 at 9 43 05 AM](https://user-images.githubusercontent.com/77470406/153691436-4878259a-894d-46fa-a9a1-7c95d628932b.png)

zzhikai committed:
Implement contact us section

Reference from FTS Month as well as FTS Website

zzhikai commented:
I think I should not include the social media icons? As the footer already has it.

jerrenss reviewed (changes requested):
Looks great! Just some changes to make:
1. Optimise the NUS_SOC image using online tools / use a smaller sized image, 1.96MB is quite large
2. Remove the icons in contact as the footer already has them

Thanks!

zzhikai committed:
Resolve comments

Reduce background image size, remove Social Media Icons, reduce height of section to 40vh

zzhikai commented:
> Looks great! Just some changes to make:
> 
> 1. Optimise the NUS_SOC image using online tools / use a smaller sized image, 1.96MB is quite large
> 2. Remove the icons in contact as the footer already has them
> 
> Thanks!

Updated in 8a7867b8450a79da2d5b88acbc694029465ccd2e:

![Screenshot 2022-02-12 at 11 58 42 PM](https://user-images.githubusercontent.com/77470406/153718424-8d01bb43-9899-4731-bebf-9c3eb2d2667e.png)


![Screenshot 2022-02-12 at 11 58 17 PM](https://user-images.githubusercontent.com/77470406/153718407-444fe234-71d7-45b7-addb-6aabf2fe884e.png)
